elastic/indexer/insta_posts/main/main.go

```diff
@@ -92,3 +101,27 @@ func upsertPost(post *post, client *elasticsearch.Client) error {
 	}
 	return nil
 }
+
+func createUpsertBody(post *post) *map[string]interface{} {
+	var commentUpsert = map[string]interface{}{
+		"script": map[string]interface{}{
+			"source": "ctx._source.caption = params.caption",
+			"lang":   "painless",
+			"params": map[string]interface{}{
+				"caption": "",
+			},
+		},
+		"upsert": map[string]interface{}{
+			"user_id": "",
+			"caption": "",
```

Comment:
You can put the `caption` and `user_id` into the map directly here, instead of modifying it below